src/commands/add/addCredential.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from './internal/input'
 import { TargetScope } from '../../types/targetScope'
 import { saveConfig } from './internal/saveConfig'
+import { HttpsAgentOptions } from '@sasjs/utils/types/httpsAgentOptions'
```

Comment:
It should be imported from `@sasjs/utils/types` like other types

Reply:
fixed.